src/JSONSchema/path-handler.ts

```diff
@@ -0,0 +1,107 @@
+import { JSONSchemaType } from './types'
+import { useFormContext } from '../components'
+
+const concatFormPath = (path: string, newNode: string): string => {
+  return path + '/' + newNode
+}
+
+const getSplitPath = (path: string): Array<string> => {
+  const split = path.split('/')
+  if (split[0] === '#') {
+    split.shift()
+  }
+  if (split[split.length - 1] === '') {
+    split.pop()
+  }
+  return split
+}
+
+const useObjectFromPath = (path: string): [JSONSchemaType, boolean, string] => {
+  const splitPath = getSplitPath(path)
+  let currentOriginal = useFormContext().schema
```

Comment:
The schema is getting typed as  `any`, I think we should fix this

Reply:
Should get right to this